Bottom: b97928cbeb59ebcda1dbb6c22e7b425913dd44fd
Top:    034690a58946f3e875a3f8548639d4c70beaacf8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-10 14:33:43 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
